MarsRover/Rover.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+
+namespace MarsRover
+{
+    public class Rover
+    {
+        // has a Direction & a Location 
+        // Move - Forward/Backward
+        // Turn - Left/Right
+        public Rover(Direction direction, int x, int y)
+        {
+            Direction = direction;
+            Location = new Location(x, y);
+        }
+
+        public Direction Direction { get; private set; }
+
+        public Location Location { get; private set; }
+
+        public Direction Turn(Command command) // TODO: talk at Mentor meeting about changing to switch expression

```

Comment:
let's refactor this class to make it easier to read but I like that you've separated turn and move